indra/newview/llviewerobject.cpp

```diff
@@ -3466,6 +3555,52 @@ BOOL LLViewerObject::loadTaskInvFile(const std::string& filename)
     return TRUE;
 }
 
+BOOL LLViewerObject::loadTaskInvLLSD(const LLSD& inv_result)
```

Comment:
Could have just return void since the result is never actually use in some way (or at least return true booleans, since the fake BOOL is on its way out with Maint-A)